src/utils.test.ts

```diff
@@ -183,6 +267,92 @@ describe('utils', () => {
         `,
       )
     })
+
+    it('should add co-authors if more than author is detected', () => {
+      const expected = getMergeData(
+        createTestPullRequest({
+          pullRequest: {
+            headRefName: 'feat/add-something',
+            baseRefName: 'master',
+            title: 'feat: Add new component',
+            body: stripIndent`
+              ## Summary
+              Add a new Component
+
+              ## Release Category
+              Components
+            `,
+            number: 1240,
+            id: '',
+            author: {
+              login: 'janesmith',
+            },
+            commits: {
+              totalCount: 2,
+              nodes: [
+                {
+                  commit: {
+                    additions: 1,
+                    deletions: 1,
+                    message: 'Fix a typo',
+                    authoredByCommitter: true,
+                    authors: {
+                      nodes: [
+                        {
+                          name: 'John Doe',
+                          email: 'john.doe@example.com',
+                          user: null,
+                        },
+                      ],
+                    },
+                  },
+                },
+                {
+                  commit: {
+                    additions: 1,
+                    deletions: 1,
+                    message: 'Fix a typo',
+                    authoredByCommitter: true,
+                    authors: {
+                      nodes: [
+                        {
+                          name: 'John Doe',
+                          email: 'john.doe@example.com',
+                          user: {
+                            login: 'JohnDoe',
+                          },
+                        },
+                        {
+                          name: 'Jane Doe',
+                          email: 'jane.doe@example.com',
+                          user: {
+                            login: 'JaneDoe',
+                          },
+                        },
+                      ],
+                    },
+                  },
+                },
+              ],
+            },
+            mergeable: 'MERGEABLE',
+            autoMergeRequest: null,
+          },
+        }),
+      )
+
+      expect(expected).toHaveProperty(
+        'commitBody',
+        stripIndent`
+          Add a new Component
+
+          [category:Components]
+
+          Co-authored-by: @JohnDoe <john.doe@example.com>
+          Co-authored-by: @JaneDoe <jane.doe@example.com>
+        `,
+      )
```

Comment:
If multiple contributors are detected outside the PR author, the co-author lines will be committed in the squashed commit message which is picked up by release notes.